cores/esp8266/cont.h

```diff
@@ -45,6 +49,12 @@ typedef struct cont_ {
         unsigned* struct_start;
 } cont_t;
 
+/* Not static, used in core_esp8266_postmortem.c.
+ * Placed into noinit section because we assign value to this variable
+ * before .bss is zero-filled, and need to preserve the value.
+ */
+extern cont_t* g_pcont __attribute__((section(".noinit")));
```

Comment:
`__attribute__` seems to be needed only on definition, not on declaration. Details about placement into noinit section can live there as well.